### ISSUE TITLE
How to get execution report  and the real-time orderbook

### ISSUE THREAD

**Issue by oss-jtyd** [mzheravin/exchange-core#91]
Hi @mzheravin
     How to get execution report  and the real-time orderbook

**Comment by mzheravin** [comment]
Hi @oss-jtyd 
Execution report can be cunstructed from matching events.
Real-time orderbook is not supported yet.

**Comment by oss-jtyd** [comment]
thanks ，can his project have a discuss group in slack or telegram？

**Comment by magick93** [comment]
Regarding the live orderbook - are there any orderbooks events that could
be handled, and used to construct a realtime orderbook?

This could also be used to update a datagrid, eg, Apache Ignite.

**Comment by mzheravin** [comment]
@oss-jtyd created a group in telegram: https://t.me/exchangecoretalks

**Comment by mzheravin** [comment]
@magick93 yes I was thinking of that, new version will have it, something similar to NASDAQ ITCH protocol.
